src/util.ts

```diff
@@ -22,6 +22,8 @@ export const DEV_DEPENDENCIES_DIR = path.join(PROJECT_CACHE_DIR, 'dev');
 const LOCKFILE_HASH_FILE = '.hash';
 
 export const HAS_CDN_HASH_REGEX = /\-[a-zA-Z0-9]{16,}/;
+export const HTML_JS_REGEX = /(<script.*?>)(.*?)<\/script>/gms;
```

Comment:
We’re not really testing this for `type="module"`, but I think that’s fine? Just thinking out loud.

Reply:
Good catch. We actually share this logic in two places, the other is when we scan Vue & Svelte files for `<script>` tags, where I don't think `type="module"` is required.